styleguide.config.js

```diff
@@ -46,6 +46,10 @@ module.exports = {
           name: 'Badge',
           components: './packages/design-system/src/components/Badge/Badge.js'
         },
+        {
+          name: 'Banner',
+          components: './packages/design-system/src/components/Banner/Banner.js'
+        },
```

Comment:
I think this should be placed under `Alerts`

Reply:
@dprzybylek ?

Reply:
moved along with other "B..." components ;)

Reply:
I feel that @quarties meant moving it under "Alerts" like it's done for Toast and ToastWrapper

Reply:
any update here?

Reply:
Yeah, I meant it to be like `Toast` and `ToastWrapper`

Reply:
done